rbd.c

```diff
@@ -943,8 +943,18 @@ static void rbd_finish_aio_generic(rbd_completion_t completion,
 {
 	struct tcmu_device *dev = aio_cb->dev;
 	struct tcmulib_cmd *tcmulib_cmd = aio_cb->tcmulib_cmd;
-	struct iovec *iovec = tcmulib_cmd->iovec;
-	size_t iov_cnt = tcmulib_cmd->iov_cnt;
+
+    struct iovec *iovec;
```

Comment:
We use linux kernel coding style when we can, so use tabs instead of spaces and the tabs must be 8 cols wide.

We also prefer to declare variables at the beginning of the function, so this chunk of code would go after the "int tmu_r" declaration in this function. It is ok to do declare variables later if they are inside a if/loop.

Reply:
I have fixed it. Thanks for your review.